src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "altinn-app-frontend",
-  "version": "3.13.6",
+  "version": "3.14.6",
```

```suggestion
  "version": "3.14.0",
```

Could potentially need to change if other pr's are merged.

---

Thats fine, I'm now used that, hehe